src/components/Header/Header.scss

```diff
@@ -0,0 +1,39 @@
+$headerHeightDesktop: 160px;
+$headerHeightMobile: 64px;
+.header {
+  display: flex;
+  flex-direction: row;
+  background-color: #fe9436;
+  justify-content: center;
+  align-items: center;
+  margin: auto 0;
+  @media screen and (max-width: 480px) {
+    height: $headerHeightMobile;
+  }
+  @media screen and (min-width: 481px) {
+    height: $headerHeightDesktop;
+  }
+  .header-title {
+    font-size: calc(14px + 2vmin);
+    flex: 1;
+    color: white;
+    text-align: center;
+    @media screen and (max-width: 480px) {
```

Comment:
Muito legal como você lidou com os break points aqui!!! Será que dá para gente melhorar esse código?

Pergunto isso porque a impressão que tenho é que as vezes esse caminho pode ser caótico.

Reply:
Thank you, i found this way in a video and adapted to our necessity!

I don't know if i can do this, but i tried to improve using @mixin in the lines that were repeated, but it gone wrong.

But i did not understand why it can be caotic, can you explain it to me? While i don't understand it, i'll try another way to do this!

Reply:
Great work lulubas! I am very proud. 
Some insights for this one:
- You are duplicating `@media screen...`, I can see 6 of them in your code. What do you think about add the class into your breakpoints instead of add the breakpoint into the classes?
- Should we really need to define two breakpoints? What do you think about having a default value (e.g desktop) and only change the values when the breakpoint change to mobile?

Let me know if you need any help. But anyway, If you don't figure out a potential solution we can discuss in the next meeting.

Reply:
Thanks for the insights João, i think i did it now the way you adviced me, so i already made the changes about the repetition of the "@media" and the definition of a default value for header height (desktop).